sunpy/map/tests/test_mapbase_dask.py

```diff
@@ -0,0 +1,67 @@
+import copy
+
+import numpy as np
+import pytest
+
+import astropy.units as u
+import astropy.wcs
+from astropy.io import fits
+from astropy.io.fits.verify import VerifyWarning
+
+import sunpy.map
+from sunpy.data.test import get_test_filepath
+
+
+def test_dask_array(generic_map):
+    dask_array = pytest.importorskip('dask.array')
```

Comment:
Can we do this for the entire file instead of in each test?

Reply:
That I do not know. You can do it with a fixture and then every thing that uses that fixture will be skipped if that fixture is skipped.

Reply:
Ok I've cleaned them up so that that `importorskip` is only in one fixture.